Update index.md (#489)
dotnet/AspNetDocs: aspnet/aspnet/overview/owin-and-katana/index.md

---
# OWIN and Katana

> Katana is a flexible set of components for building and hosting Open Web Interface for .NET (OWIN)-based web apps. New development should use [ASP.NET Core](/aspnet/core/introduction-to-aspnet-core). ASP.NET Core is an evolution of `Microsoft.Owin.*`.

Reviewer: Do we want to be even more explicit?
```suggestion
Katana is a flexible set of components for building and hosting Open Web Interface for .NET (OWIN)-based web apps. New development should use [ASP.NET Core](/aspnet/core/introduction-to-aspnet-core). The `Microsoft.Owin.*` libraries are not available for .NET Core because ASP.NET Core has equivalent replacements for them.
```

> Katana is a flexible set of components for building and hosting Open Web Interface for .NET (OWIN)-based web apps. New development should use [ASP.NET Core](/aspnet/core/introduction-to-aspnet-core). ASP.NET Core  is an evolution of `Microsoft.Owin.*`.

Author: ```suggestion
Katana is a flexible set of components for building and hosting Open Web Interface for .NET (OWIN)-based web apps. New development should use [ASP.NET Core](/aspnet/core/introduction-to-aspnet-core). ASP.NET Core is an evolution of `Microsoft.Owin.*`.
```

> Katana is a flexible set of components for building and hosting Open Web Interface for .NET (OWIN)-based web apps. New development should use [ASP.NET Core](/aspnet/core/introduction-to-aspnet-core). ASP.NET Core ASP.NET Core is an evolution of `Microsoft.Owin.*`.

Author: ```suggestion
Katana is a flexible set of components for building and hosting Open Web Interface for .NET (OWIN)-based web apps. New development should use [ASP.NET Core](/aspnet/core/introduction-to-aspnet-core). ASP.NET Core  is an evolution of `Microsoft.Owin.*`.
```
